src/libPMacc/include/particles/memory/frames/Frame.hpp

```diff
@@ -149,6 +154,12 @@ public InheritLinearly<typename T_ParticleDescription::MethodsList>,
         return std::string(boost::mpl::c_str<Name>::value);
     }
 
+    /* \todo find a generic solution to add `previousFrame` and `nextFrame`
+     * only if we use this frame in a double linked list
```

Comment:
Do you mean:

**use this part of the code** only if we use this frame in a double linked list